di-ipv-credential-issuer-stub/src/main/resources/data/criStubEvidencePayloads.json

```diff
@@ -443,7 +443,7 @@
         "type": "IdentityCheck",
         "strengthScore": 2,
         "validityScore": 0,
-        "ci": ["QQQ"],
+        "ci": ["D02"],
```

Comment:
There are two codes in the ticket, do we need a payload for each one, or both in one, or neither?

Reply:
It's optional between both of them, so if anything it would be to have a second payload with the other code

Reply:
It's fine just to have the one as an example (just to correct the QQQ placeholder)